recipes/oatk/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: oatk
+  version: 1.0
+
+source:
+  url: https://github.com/c-zhou/oatk/archive/refs/tags/1.0.tar.gz
+  sha256: 5d1a1f44750dac04b436b99f6877b531b19e2e87a3eab956627fbb5e70af463c
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('oatk', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+  host:
+    - zlib  
+  run:
+    - zlib
+    - hmmer ==3.4
```

Comment:
Is it necessary to pin hmmer like this? With this, oatk cannot be installed into an environment where another package has a different requirement although it may actually work. Maybe `>=3.4` is enough?

Reply:
No problem!